code/_onclick/item_attack.dm

```diff
@@ -14,6 +14,16 @@
 
 /mob/living/attackby(obj/item/I, mob/user, params)
 	user.changeNext_move(CLICK_CD_MELEE)
+
+	if((butcher_results) && (stat == DEAD))
```

Comment:
@Crazylemon64 I need to move this to another spot, its not doing butchering now...it should but its not.